test/baselines/generated-reference/namespaces.html

```diff
@@ -44,6 +44,8 @@ <h1><span class="secnum">1.1</span> Clause 1.1</h1>
 </emu-production>
 </emu-grammar>
 
+    <!-- FIXME: Shouldn't this be allowed by the preceding `namespace=clause`? -->
```

Comment:
I wouldn't think so - line 53 has an algorithm with an identical AOID, and is within the same namespace.